vendor/ember/shims.js

```diff
@@ -0,0 +1,247 @@
+        'instrument':   Ember.Instrumentation.instrument,
+        'reset':        Ember.Instrumentation.reset,
+        'subscribe':    Ember.Instrumentation.subscribe,
+        'unsubscribe':  Ember.Instrumentation.unsubscribe
+      },
+      'ember-locations/hash': {
+        'default': Ember.HashLocation
+      },
+      'ember-locations/history': {
+        'default': Ember.HistoryLocation
+      },
+      'ember-locations/none': {
+        'default': Ember.NoneLocation
+      },
+      'ember-map': {
+        'default':      Ember.Map,
+        'withDefault':  Ember.MapWithDefault
+      },
+      'ember-metal/destroy': {
+        'default': Ember.destroy
+      },
+      'ember-metal/events': {
+        'addListener':    Ember.addListener,
+        'removeListener': Ember.removeListener,
+        'send':           Ember.sendEvent
+      },
+      'ember-metal/get': {
+        'default': Ember.get
+      },
+      'ember-metal/mixin': {
+        'default': Ember.Mixin
+      },
+      'ember-metal/observer': {
+        'default':        Ember.observer,
+        'addObserver':    Ember.addObserver,
+        'removeObserver': Ember.removeObserver
+      },
+      'ember-metal/on-load': {
+        'default':  Ember.onLoad,
+        'run':      Ember.runLoadHooks
+      },
+      'ember-metal/set': {
+        'default':        Ember.set,
+        'setProperties':  Ember.setProperties,
+        'trySet':         Ember.trySet
+      },
+      'ember-metal/utils': {
+        'aliasMethod':  Ember.aliasMethod,
+        'assert':       Ember.assert,
+        'cacheFor':     Ember.cacheFor,
+        'copy':         Ember.copy
+      },
+      'ember-object': {
+        'default': Ember.Object
+      },
+      'ember-platform': {
+        'assign':         Ember.merge,
+        'create':         Ember.create,
+        'defineProperty': Ember.platform.defineProperty,
+        'hasAccessors':   Ember.platform.hasPropertyAccessors,
+        'keys':           Ember.keys
+      },
+      'ember-route': {
+        'default': Ember.Route
+      },
+      'ember-router': {
+        'default': Ember.Router
+      },
+      'ember-runloop': {
+        'default':      Ember.run,
+        'begin':        Ember.run.begin,
+        'bind':         Ember.run.bind,
+        'cancel':       Ember.run.cancel,
+        'debounce':     Ember.run.debounce,
+        'end':          Ember.run.end,
+        'join':         Ember.run.join,
+        'later':        Ember.run.later,
+        'next':         Ember.run.next,
+        'once':         Ember.run.once,
+        'schedule':     Ember.run.schedule,
+        'scheduleOnce': Ember.run.scheduleOnce,
+        'throttle':     Ember.run.throttle
+      },
+      'ember-service': {
+        'default': Ember.Service
+      },
+      'ember-service/inject': {
+        'default': Ember.inject.service
+      },
+      'ember-set/ordered': {
+        'default': Ember.OrderedSet
+      },
+      'ember-string': {
```

Comment:
I don't think so, it's provided by https://github.com/stefanpenner/ember-inflector